external-crates/move/crates/move-package-alt/tests/data/lockfile_parsing_valid/basic/Move.snap

```diff
@@ -1,4 +1,4 @@
ency { local: "../foo" }) } }}, unpinned: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 463..466, value: "bar" } }: Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 469..496, value: External(ExternalDependency { r: Table({"mvr": String("@protocol/bar")}), fields: {} }) } }, Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 415..418, value: "foo" } }: Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 421..462, value: Local(LocalDependency { local: "../foo" }) } }} }, dep_overrides: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1606..1613, value: "mainnet" } }: UnpublishedDependencies { pinned: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1622..1625, value: "foo" } }: Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1628..1653, value: Git(GitDependency { repo: "", rev: "sha", path: None, phantom: PhantomData<move_package_alt::dependency::Pinned> }) } }, Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1693..1696, value: "std" } }: Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1699..1724, value: Git(GitDependency { repo: "", rev: "sha", path: None, phantom: PhantomData<move_package_alt::dependency::Pinned> }) } }}, unpinned: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1829..1832, value: "foo" } }: Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1835..1860, value: Git(GitDependency { repo: "", rev: "rev", path: None, phantom: PhantomData<move_package_alt::dependency::Unpinned> }) } }} }} }, published: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 509..516, value: "mainnet" } }: Publication { metadata: (), dependencies: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1036..1039, value: "std" } }: Git(GitDependency { repo: "", rev: "mainnet_sha", path: None, phantom: PhantomData<move_package_alt::dependency::Pinned> })} }, Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1088..1095, value: "testnet" } }: Publication { metadata: (), dependencies: {Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1498..1501, value: "std" } }: Git(GitDependency { repo: "", rev: "testnet_sha", path: None, phantom: PhantomData<move_package_alt::dependency::Pinned> }), Located { file: "tests/data/lockfile_parsing_valid/basic/Move.lock", value: Spanned { span: 1538..1541, value: "sui" } }: Git(GitDependency { repo: "", rev: "testnet_sha", path: None, phantom: PhantomData<move_package_alt::dependency::Pinned> })} }} }
```

Comment:
I think we really need to render these errors for these snapshots to make any sense (and it will hopefully become instantly clear if we've gotten the right ranges)